qa/rpc-tests/wallet_sapling.py

```diff
@@ -204,36 +205,44 @@ def run_test(self):
         # Check z_sendmanywithchangetosender
         # send from node 0 taddr to node 2 single taddr
         senderaddr = self.nodes[0].getnewaddress()
```

Comment:
I just can not get idea of changes. probably should be next?
senderaddr = self.nodes[2].getnewaddress()

Reply:
what do you mean? And what changes you are requesting?

Reply:
```
        # Check z_sendmanywithchangetosender
        # send from node 0 taddr to node 2 single taddr
```

Reply:
Yes this doesn’t seem like a helpful comment…

Reply:
It is clear now,